README.MD

```diff
@@ -1,3 +1,8 @@
 ### GIT
 
-This is the first commit for git main branch
\ No newline at end of file
+This is the first commit for git main branch
+
+### Feature-1 content
```

Comment:
This line is not required, Please mentioned why this is needed.

Reply:
Yes this is mandatory for continue